lab_5_scrapper/scrapper.py

```diff
@@ -204,6 +323,12 @@ def prepare_environment(base_path: Union[pathlib.Path, str]) -> None:
     Args:
         base_path (Union[pathlib.Path, str]): Path where articles stores
     """
+    assets_path = pathlib.Path(base_path) / 'ASSETS_PATH'
+    if not assets_path.exists():
+        assets_path.mkdir(parents=True, exist_ok=True)
```

Comment:
if it does not exist, then why do we need flag `exist_ok=True`?

Reply:
fixed

Reply:
no